quasar.conf.js

```diff
@@ -16,6 +16,9 @@ const _ = require('lodash')
 const glob = require('glob')
 const pMap = require('p-map')
 
+const netlifyServe = require('./netlify/ci/serve')
+const useNetlifyLambda = process.env.NETLIFY_LAMBDA === 'true'
```

Comment:
`serveNetlifyLambda` may be a better name since we specifically _serve_ functions in dev environment, unlike production where we _use_ Lambda too, relying on Netlify infrastructure.